src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -295,7 +295,7 @@
         "&fProduces &8Scrap &ffrom anything",
         "",
         LoreBuilderDynamic.powerBuffer(Recycler.CAPACITY),
-        LoreBuilderDynamic.powerPerTick(Recycler.ENERGY_CONSUMPTION)
+        LoreBuilderDynamic.powerPerSecond(Recycler.ENERGY_CONSUMPTION)
```

Comment:
It should say power per tick

Reply:
My last pr to extrautils literally flipped these, the naming was off since you want the display to show j/t instead of j/s

Reply:
But now they say power per tick

Reply:
Yeah thats what its supposed to be?

Reply:
They say power per sec* sorry lorebuilderdynamic.powerpersec

Reply:
Oh i see this isnt the solar panel

Reply:
Wait so you want every machine to have a J/t display, not seconds?